javatests/arcs/android/storage/service/StorageServiceEndToEndTest.kt

```diff
@@ -92,13 +104,61 @@ class StorageServiceEndToEndTest {
         DriverFactory.clearRegistrations()
     }
 
+    @Test
+    fun writeThenRead_nullInlines_inCollection_onDatabase() = runBlocking {
+        val handle = createCollectionHandle(databaseKey)
+        val entity = emptyEntityForTest()
+
+        handle.dispatchStore(entity)
+
+        val handle2 = createCollectionHandle(databaseKey)
+        val data = handle2.fetchAll()
+        assertThat(data.size).isEqualTo(1)
+        assertThat(data.toList()[0]).isEqualTo(entity)
```

Comment:
these 3 lines can be written as just
assertThat(handle2.fetchAll()).containsExactly(entity)

alternatively 
data.toList()[0] === data.first()
and data.only() will give you the first element and throw if there are more then one